openfl-tutorials/interactive_api/medmnist_fl_3D_WIP/README.md

```diff
@@ -0,0 +1,64 @@
+# MedMNIST 2D Classification Tutorial
+
+![MedMNISTv2_overview](https://raw.githubusercontent.com/MedMNIST/MedMNIST/main/assets/medmnistv2.jpg)
+
+For more details, please refer to the original paper:
+**MedMNIST v2: A Large-Scale Lightweight Benchmark for 2D and 3D Biomedical Image Classification** ([arXiv](https://arxiv.org/abs/2110.14795))
+
+
+### 1. MedMNIST Installation and Requirements
+Setup the required environments and install `medmnist` as a standard Python package from [PyPI](https://pypi.org/project/medmnist/):
+
+    pip install medmnist
+
+Or install from source:
+
+    pip install --upgrade git+https://github.com/MedMNIST/MedMNIST.git
+
+Check whether you have installed the latest [version](medmnist/info.py):
+
+    >>> import medmnist
+    >>> print(medmnist.__version__)
+
+The code requires only common Python environments for machine learning. Basically, it was tested with
+* Python 3 (>=3.6)
+* PyTorch\==1.3.1
+* numpy\==1.18.5, pandas\==0.25.3, scikit-learn\==0.22.2, Pillow\==8.0.1, fire, scikit-image
```

Comment:
Please add requirements.txt file and install medmnist along with these dependencies if not already installed with OpenFL.

Reply:
The dependencies will automatically resolve as soon as "pip install medmnist" is ran. Thanks